README.md

```diff
@@ -270,6 +270,7 @@ loadProfile.scale() // returns a LoadProfileScaler instance (documented below)
 loadProfileScaler.to(scale) // returns a scaled load profile scaled to the provided number
 loadProfileScaler.toTotalKwh(totalKwh) // returns a scaled load profile scaled based on load to the provided kwh
 loadProfileScaler.toAverageMonthlyBill(amount, rate) // returns a scaled load profile scaled to the amount based on the provided RateInterface, rate
+loadProfileScaler.toMonthlyKwh(monthlyKwh) // returns a scaled load profile where the .sumByMonth() method returns an array equivalent to the given `monthlykWh` array
```

Comment:
how is this description? is there a clearer way to see this?

Reply:
I took a stab at it... but it's perfectly fine as is.

> returns a scaled load profile where each month is individually scaled to the `monthlyKwh`, where `monthlyKwh` is an array of 12 months.